packages/infra/lib/rate-limiting-nested-stack.ts

```diff
@@ -1,18 +1,29 @@
 import { Duration, NestedStack, NestedStackProps } from "aws-cdk-lib";
 import { SnsAction } from "aws-cdk-lib/aws-cloudwatch-actions";
-import * as ec2 from "aws-cdk-lib/aws-ec2";
 import * as dynamodb from "aws-cdk-lib/aws-dynamodb";
 import * as cloudwatch from "aws-cdk-lib/aws-cloudwatch";
 import { Construct } from "constructs";
 import { EnvConfig } from "../config/env-config";
-import { LambdaLayers } from "./shared/lambda-layers";
 import { isProd } from "./shared/util";
 
 interface RateLimitingNestedStackProps extends NestedStackProps {
   config: EnvConfig;
-  vpc: ec2.IVpc;
   alarmAction?: SnsAction;
-  lambdaLayers: LambdaLayers;
+}
+
+function getSettings(props: RateLimitingNestedStackProps) {
+  return {
+    ...props,
+    dynamoConstructName: "APIRateLimit",
+    dynamoRateLimitPartitionKey: "cxIdAndOperationAndWindow",
+    dynamoReplicationRegions: isProd(props.config) ? ["us-east-1"] : ["ca-central-1"],
+    dynamoReplicationTimeout: Duration.hours(3),
+    dynamoPointInTimeRecovery: true,
+    consumedWriteCapacityUnitsAlarmThreshold: 1000,
+    consumedWriteCapacityUnitsAlarmPeriod: 1,
+    consumedReadCapacityUnitsAlarmThreshold: 1000,
+    consumedReadCapacityUnitsAlarmPeriod: 1,
+  };
 }
```

Comment:
😍 

Don't we want to have diff alarm thresholds and replication settings per env?

Also, can you double check if we really have to use ca-central-1 for non-prod replication? Thinking that it would be good to avoid having stuff on regions that we don't use - NABD and we can chat on Slack.

Reply:
Updated alarms.

Confirmed replication is working... there's some clean up we can do for DDB (empty tables on staging) but per slack, let's leave replication as is.